src/Umbraco.Core/Packaging/InstallationSummary.cs

```diff
@@ -32,6 +32,7 @@ public InstallationSummary(string packageName)
         public IEnumerable<IPartialView> PartialViewsInstalled { get; set; } = Enumerable.Empty<IPartialView>();
         public IEnumerable<IContent> ContentInstalled { get; set; } = Enumerable.Empty<IContent>();
         public IEnumerable<IMedia> MediaInstalled { get; set; } = Enumerable.Empty<IMedia>();
+        public IEnumerable<EntityContainer> EntityContainersInstalled { get; set; } = Enumerable.Empty<EntityContainer>();
```

Comment:
Can we change this to use `ITreeEntity` instead, as all other properties expose interface types as well?

Reply:
Looking at the services, they use the concrete type as well indeed:
https://github.com/umbraco/Umbraco-CMS/blob/5bfab13dc5a268714aad2426a2b68ab5561a6407/src/Umbraco.Core/Services/IContentTypeServiceBase.cs#L82
https://github.com/umbraco/Umbraco-CMS/blob/5bfab13dc5a268714aad2426a2b68ab5561a6407/src/Umbraco.Core/Services/IDataTypeService.cs#L21
https://github.com/umbraco/Umbraco-CMS/blob/5bfab13dc5a268714aad2426a2b68ab5561a6407/src/Umbraco.Core/Persistence/Repositories/IEntityContainerRepository.cs#L5

Might make sense to add an `IEntityContainer` abstraction, update the services and after that also this property. These are all breaking changes, so not something for now, but maybe the next major?

Reply:
Perhaps start and share a document we can populate as we go for V10?  I'll leave this as is for this PR though then.